ecs_logging/_stdlib.py

```diff
@@ -209,7 +209,12 @@ def format_to_ecs(self, record):
                 continue
             value = extractors[field](record)
             if value is not None:
-                merge_dicts(de_dot(field, value), result)
+                # special case ecs.version that should not be de-dotted
```

Comment:
It looks like log.level survives de_dot but I can't see where it gets a different treatment than the rest

Reply:
weird! 🤔